chcemvediet/apps/inforequests/tests/test_cron.py

```diff
@@ -95,7 +95,7 @@ def test_inforequest_last_undecided_email_reminder_is_updated_if_remider_is_sent
         message_set = self._call_cron_job()
 
         inforequest = Inforequest.objects.get(pk=inforequest.pk)
-        self.assertAlmostEqual(inforequest.last_undecided_email_reminder, local_datetime_from_local(u'2010-10-20 10:33:00'), delta=datetime.timedelta(seconds=10))
+        self.assertAlmostEqual(inforequest.last_undecided_email_reminder, local_datetime_from_local(u'2010-10-20 10:33:00'), delta=datetime.timedelta(milliseconds=300))
```

Comment:
@martinmacko47 Pustal som testy viac krat, cas by mal byt nastaveny dobre, ak by v buducnosti padali testy kvoli casom, tak to poom mozeme zmenit.

Reply:
:ok: